sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -305,7 +326,7 @@ def has_simple_output_type(self):
     return (self.output_types and len(self.output_types[0]) == 1 and
             not self.output_types[1])
 
-  def strip_iterable(self):
+  def strip_iterable(self):  # type: (...) -> IOTypeHints
```

Comment:
Type comment should go after the function.  No need for `...` unless you are skipping the definition of an argument (the type of `self` is implicit). 

```suggestion
  def strip_iterable(self):
    # type: () -> IOTypeHints
```